### ISSUE TITLE
[Accessibility] No alert message is spoken to screen reader when review start action is triggered

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode-pull-request-github#5526]
Type: <b>Bug</b>

CC @meganrogge 

1. Open any PR where you are assigned as a reviewer.

1. Start reviewing. Just FYI, I was not able to find a way to start review in the PR webview so I assigned the keybinding on my end:

```json
[
  {
    "key": "ctrl+shift+g r",
    "command": "pr.startReview"
  }
]
```

1. See if you have any non-visual hint as to whether the review action has started, such as aria alert "Review started."

Extension version: 0.77.2023112911
VS Code version: Code - Insiders 1.85.0-insider (8762127fc97b8aaae5f3a0dd6a6d853d9a1b7574, 2023-11-29T17:38:23.440Z)
OS version: Windows_NT x64 10.0.22631
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|11th Gen Intel(R) Core(TM) i5-1145G7 @ 2.60GHz (8 x 2611)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|15.71GB (5.53GB free)|
|Process Argv|--crash-reporter-id b05b88e5-8894-4031-ae34-fa034ebddea9|
|Screen Reader|yes|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv695:30137379
vsins829:30139715
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstes627:30244334
vslsvsres303:30308271
pythontb:30258533
pythonptprofiler:30281269
vshan820:30294714
vscod805cf:30301675
bridge0708:30335490
bridge0723:30353136
vsaa593:30376534
pythonvs932:30404738
py29gd2263:30784851
vsclangdf:30492506
c4g48928:30535728
dsvsc012:30540252
pynewext54:30618038
a9j8j154:30646983
showlangstatbar:30737417
pythonfmttext:30716741
fixshowwlkth:30771523
showindicator:30805243
pythongtdpath:30726887
i26e3531:30792625
welcomedialog:30812478
pythonnosmt12:30779711
pythonidxpt:30768918
pythonnoceb:30776497
asynctok:30898717
dsvsc013:30777762
dsvsc014:30777825
pythonmpswarning:30901777
dsvsc015:30821418
pythontestfixt:30866404
pythonregdiag2:30902439
pyreplss1:30879911
pythonmypyd1:30859725
pythoncet0:30859736
pythontbext0:30879054
accentitlementst:30870582
dsvsc016:30879898
dsvsc017:30880771
dsvsc018:30880772
aa_t_chat:30882232

```

</details>

**Comment by jooyoungseo** [comment]
No alert message is spoken, either when I approved the PR. Need some hint like "PR approved."

**Comment by alexr00** [comment]
> Start reviewing. Just FYI, I was not able to find a way to start review in the PR webview so I assigned the keybinding on my end:

Just like on github.com, you start a review by leaving a comment. No need to run an extra command. Is an alert still needed for this?

> No alert message is spoken, either when I approved the PR. Need some hint like "PR approved."

I'm trying to add an alert for this, but I'm having trouble. @meganrogge, maybe you can help:
- Is there a way to just make an alert be read? I couldn't find one, which lead me to the next point.
- I've tried to tie an alert to some HTML element that get's shown and then added `role="alert"`. This has 2 issues that I can see
  - I have to guess when the right to alert is since the UI element can show even when the review has not just been submitted. My guess is 1 minute from when the review was submitted.
  - If the PR was already just approved and the alert was read, then you approve again, the alert doesn't seem to get read the second time.

**Comment by connor4312** [comment]
This doesn't work for me with NVDA on the latest prerelease version of GHPR. I use the keyboard to action the "Start Review" button in a comment, but then focus just moves back to the editor and it reads out the editor title without announcing the review was started.

**Comment by meganrogge** [comment]
@alexr00 , sorry - I am just seeing this. 

You should check out what we do in core to get an `alert` to work. It is a little complicated. 

https://github.com/microsoft/vscode/blob/c4ea2a881dea86cf90c17fa785db1d7288c52bdb/src/vs/base/browser/ui/aria/aria.ts#L45-L60

I am not sure we need an alert here - are there cases when a review start action will fail? If not, then the user should expect this action to succeed and thus IMO doesn't need an alert to indicate that has happened.

**Comment by alexr00** [comment]
I added an alert when the review is submitted, not when the review is started. To verify:
- Install the pre-release version of the extension
- Checkout a PR
- Open the PR description. 
- Turn on the screen reader of your choice.
- Leave a comment and submit your review by either commenting, requesting changes, or approving. Verify that you hear "commented on pull request", "changes requested on pull request", or "pull request approved" depending on which you choose.

**Comment by meganrogge** [comment]
@alexr00 this works well 👏🏼
